src/components/Parcels/Parcels.jsx

```diff
@@ -198,7 +198,7 @@ const Parcels = () => {
         subHeaderComponent={buildToolBar()}
         selectableRows
         pointerOnHover
-        noDataComponent={'אין חבילות להצגה'}
+        noDataComponent={parcelExtendedData.error ? 'אופס משהו השתבש, פנה אל מנהל המערכת' : 'אין חבילות להצגה'}
```

Comment:
should not it be the opposite ? when error display 'oops...'  otherwise (if it is just null)  display 'nothing to show'

Reply:
It is indeed like you say, it's just that the hebrew messes up the line and make it look like it's the opposite.

Reply:
maybe using constants will help in this mixed languages case.

Reply:
Done.